src/Command.js

```diff
@@ -18,7 +18,11 @@ export default class Command {
     this.repository = new Repository();
     this.progressBar = progressBar;
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
-    this.toposort = !flags || (flags.sort == null ? true : flags.sort);
+
+    const {sort} = this.getOptions();
+
+    // If the option isn't present then the default is to sort.
+    this.toposort = sort == null || sort;
```

Comment:
Since `this.toposort` is never referenced until the `initialize` method of subclasses, it seems like we could wait to determine this in `runPreparations()` or something. My concern (perhaps unfounded?) is that calling getOptions() in the constructor might "miss" the `otherCommandConfigs` override? Do I just suck at class inheritance?